### DISCUSSION TITLE
regarding showImagePicker

### DISCUSSION THREAD

**Discussion by gsin1617** [react-native-image-picker/react-native-image-picker#1609] [Q&A]
as showImagePicker api is removed can someone guide me how to I create those options do I need to create them manually ?

**Comment by Johan-dutoit** [comment]
Yes, you need to create them manually.  My suggestion, is either use a [bottom sheet](https://github.com/gorhom/react-native-bottom-sheet) or [action sheet](https://github.com/expo/react-native-action-sheet).
